### PR TITLE
Scaling

### PR DESCRIPTION
**Pull request by AsaCooperStickland**
make the project pip installable with 'poetry', and add a way to run a series of experiments and install the results, `src/main.py`.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    # suppress debug warnings from the Requests library
+    logging.getLogger("urllib3").setLevel(logging.WARNING)
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(
+        description="Run model sizes and get losses for texts in a file"
+    )
+    parser.add_argument(
+        "--dataset-name",
```
Comment (mbalesni) [inline src/main.py]: BIG-Bench uses “task” instead of dataset, let’s stay consistent to it?

File: src/main.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        type=str,
```
Comment (mbalesni) [inline src/main.py:65]: Also, how about supporting multiple tasks? Leave for next PR?

**Review by mbalesni — CHANGES_REQUESTED** [review]
Awesome stuff! Let’s do `dataset-name` —> task and optionally several tasks?

**Comment by AsaCooperStickland** [comment]
made the suggested changes!

**Comment by mbalesni** [comment]
I wanted to add summary performance plots, but will leave it for later and merge now; it's been way too long!

**Pull request merged**